packages/teams-js/src/public/tasks.ts

```diff
@@ -127,7 +127,7 @@ export namespace tasks {
 
   /**
    * Sets the height and width of the {@link TaskInfo} object to the original height and width, if initially specified,
-   * otherwise uses the height and width values corresponding to {@link TaskModuleDimension | TaskModuleDimension.Small}
+   * otherwise uses the height and width values corresponding to {@link DialogDimension | TaskModuleDimension.Small}
```

Comment:
This is kind of funny and maybe it's just a docs issue. It seems like `TaskModuleDimension` is still defined and exported:
https://github.com/OfficeDev/microsoft-teams-library-js/blob/993debfdc0dfba68bdf3126bc936627c210cc5bd/packages/teams-js/src/public/constants.ts#L100
...but it's really just an alias (re-export) of the `DialogDimension` type. Do re-exports like this not show up in documentation?